srcalc/src/test/java/gov/va/med/srcalc/service/DefaultCalculationServiceTest.java

```diff
@@ -78,10 +92,32 @@ public final void testSetInvalidSpecialty() throws InvalidIdentifierException
         final int PATIENT_DFN = 1;
         
         // Create the class under test.
-        final DefaultCalculationService s = defaultCalculationService();
+        final DefaultCalculationService s = new DefaultCalculationService(
+                fMockSpecialtyDao, fPatientDao, fSurgeryDao);
```

Comment:
Why not have a method for this, since you repeated the constructor so many times?